.github/workflows/ci-aarchxx-cross.yml

```diff
@@ -108,9 +108,8 @@ jobs:
         DISABLE_FORMAT_CHECKS: ${{ steps.are_format_checks_disabled.outputs.disable_format_checks }}
 
   # ARM cross-compile with gcc, with some tests run under QEMU.
```

Comment:
PR title nit:

> i#7270-i#i5437

s/-i#i/,i#/

Reply:
Done.  (Maybe should have just picked one; two is not really covered in our style guide/examples.)

Reply:
> s/-i#i/,i#/

Looks like the duplicate `i` preceding the second issue number was accidentally left unchanged.

Reply:
Whoops, missed that.